### DISCUSSION TITLE
Disable error messages for not found images

### DISCUSSION THREAD

**Discussion by kodmanyagha** [vercel/next.js#66368] [Help]
### Summary

Hi folks, I'm getting that error for not found images:

```
 ⨯ The requested resource isn't a valid image for /crypto-icons/128/icon/phb.png received text/html; charset=utf-8
 GET /crypto-icons/128/icon/nfp.png 404 in 861ms
```

I want to disable these messages because this is wasting my console outputs. I already know these images not exist and I don't want to see these continuously. How can I do that? Thanks.

Edit: I seached google but I didn't find any solution neighter in google nor nextjs site.

**Comment by serdar-webdev** [comment]
If you need to disable images error logs from certain page, so you can use middleware fot temprary solution;

const url = new URL(request.url);

  if (url.href.endsWith(".jpg")) {
    return NextResponse.redirect("http://localhost:3000/test");
  }
  
  like this in middleware, you will get all req to server with .jpg extension and redirects response to its page

**Reply by kodmanyagha** [reply]
Thanks for reply. I think about this before but I'm looking for real solution.

**Reply by kodmanyagha** [reply]
I tried your suggestion. Now I'm getting more annoying error:

```
ImageError: Unable to optimize image and unable to fallback to upstream image
    at imageOptimizer (node_modules/next/dist/server/image-optimizer.js:688:19)
    at async cacheEntry.imageResponseCache.get.incrementalCache (node_modules/next/dist/server/next-server.js:182:65)
    at async node_modules/next/dist/server/response-cache/index.js:90:36
    at async node_modules/next/dist/lib/batcher.js:45:32 {
  statusCode: 400
```

**Reply by serdar-webdev** [reply]
![code](https://github.com/vercel/next.js/assets/115923465/09f6dbe0-5f37-46a6-8b9f-5ad06254d3cd)
![code-1](https://github.com/vercel/next.js/assets/115923465/0f7c3917-a403-41b1-86d3-55944967a892)

here is my example, i didnt get any errors while building, development and production mode;
 "next": "14.1.0"

**Reply by serdar-webdev** [reply]
Here is another way, https://nextjs.org/docs/app/api-reference/next-config-js/redirects#regex-path-matching;
  implementing redirects in next config maybe it will match your way
